libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1069,6 +1071,17 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         }
     }
 
+    /**
+     * Set [RerouteOptionsAdapter]. It allows modify [RouteOptions] of default implementation of
+     * [NavigationRerouteController] ([RerouteController])
```

Comment:
```suggestion
     * Set [RerouteOptionsAdapter]. It allows to modify [RouteOptions] before a reroute request is sent if the default reroute controller is used. See [defaultRerouteController]. Pass `null` to clear the adapter.
```

Reply:
`defaultRerouteController` is private. I would remove this string

Reply:
:+1: Let's drop the "See [defaultRerouteController]." from the suggestion.